src/rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(data, book, person)
+    @data = data
```

Comment:
- [ ] Based on your implementation this should be `date`, not `data`. Kindly fix that.